keystone.js

```diff
@@ -4,26 +4,13 @@ require('dotenv').config();
 
 // Require keystone
 var keystone = require('keystone');
-var handlebars = require('express-handlebars');
 
+var keystoneInit = require('./config.js').keystoneInit;
```

Comment:
would read cleaner as `require('./config.js')`

Reply:
might also be good to rename this as `keystoneConfig`

Reply:
`config` instead of `keystoneConfig` with the intent of adding more config in the same file.